<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calendar</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="loginButton">
              <property name="styleSheet">
               <string notr="true">font: 87 14pt &quot;SF Pro&quot;;</string>
              </property>
              <property name="text">
               <string>Login</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="createCalendarButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
                </property>
                <property name="text">
                 <string>Create</string>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="shareCalendarButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
                </property>
                <property name="text">
                 <string>Share</string>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_2"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Current calendar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_calendar">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Calendar organizer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_organizer">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Current user</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_user">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCalendarWidget" name="calendarWidget"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Lohit Telugu&quot;;
font: 14pt &quot;Sans Serif&quot;;</string>
            </property>
            <property name="text">
             <string>ToDo</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_createTodo">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Create a Todo</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Lohit Telugu&quot;;
font: 14pt &quot;Sans Serif&quot;;</string>
            </property>
            <property name="text">
             <string>Events</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_Events"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_createEvent">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Create a Event</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
